oceanwaters/doc/setup_dev_env.md

```diff
@@ -181,27 +181,25 @@ contents can be found [here](https://github.com/nasa/GSAP/wiki/Getting-Started).
 
 ### ROS
 
-Installation of OceanWATERS requires prior installation of ROS Melodic. ROS
+Installation of OceanWATERS requires prior installation of ROS Noetic. ROS
 binary downloads for Ubuntu Linux are available at
 [http://wiki.ros.org/ROS/Installation](http://wiki.ros.org/ROS/Installation),
 and instructions for installing ROS are available at
-[http://wiki.ros.org/melodic/Installation/Ubuntu](http://wiki.ros.org/melodic/Installation/Ubuntu).
+[http://wiki.ros.org/noetic/Installation/Ubuntu](http://wiki.ros.org/noetic/Installation/Ubuntu).
 
-* Install ROS (Melodic version) by following
-[these instructions](http://wiki.ros.org/melodic/Installation/Ubuntu). Select the
-ros-melodic-desktop-full package when you get to that step.
+* Install ROS (Noetic version) by following
+[these instructions](http://wiki.ros.org/noetic/Installation/Ubuntu). Select the
+ros-noetic-desktop-full package when you get to that step.
 
 By default, ROS is installed in `/opt/ros/release`. In the remainder of this document,
```

Comment:
I don't know who wrote this but I think it is rather confusing .. is ROS installed in `/opt/ros/release/ROS`? I think it is better be re-phrased. Maybe something like:

"If you follow the instructions as provided by the ROS installation guide then ROS noetic will be deployed to `/opt/ros/noetic`, which is the path we are going to assume for the reminder of this guide."

Reply:
Made an equivalent edit.